CHANGELOG.md

```diff
@@ -4,16 +4,18 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ## Next
 
-### Added
+### Changed
 
-- Allow to pass Cloud authentication token via TUIST_CLOUD_TOKEN even when not CI [#3380](https://github.com/tuist/tuist/pull/3380) by [@danyf90](https://github.com/danyf90)
-- Support for cache categories argument in `tuist clean` command [#3407](https://github.com/tuist/tuist/pull/3407) by [@danyf90](https://github.com/danyf90)
+- **Breaking** Minimum supported Xcode version bumped to 12.4.
 
 ### Fixed
-
-- Fix caching of manifests that use plugins [#3370](https://github.com/tuist/tuist/pull/3370) by [@luispadron](https://github.com/luispadron)
-- Fix linking of transitive precompiled static frameworks for unit test with a host app [#3357](https://github.com/tuist/tuist/pull/3357) by [@adellibovi](https://github.com/adellibovi), [@kwridan](https://github.com/kwridan)
 - settings-to-xcconfig migration command produces correct string format. [#3260](https://github.com/tuist/tuist/3260) by [@saim80](https://github.com/saim80)
+- Fix caching of manifests that use plugins [#3370](https://github.com/tuist/tuist/pull/3370) by [@luispadron](https://github.com/luispadron)
+
+### Added
+
+- Allow to pass Cloud authentication token via TUIST_CLOUD_TOKEN even when not CI [#3380](https://github.com/tuist/tuist/pull/3380) by [@danyf90](https://github.com/danyf90)
+- Support for cache categories argument in `tuist clean` command [#3407](https://github.com/tuist/tuist/pull/3407) by [@danyf90](https://github.com/danyf90)
```

Comment:
Should we keep an order of section described [here](https://keepachangelog.com/en/1.0.0/)?

```
Added
Changed
Deprecated
Removed
Fixed
Security
```

We point https://keepachangelog.com/en/1.0.0/ website on a very top of this file.

Reply:
good point, in this chase the **Breaking** change looks most important to me and therefore I would add it on top. but this are just my 2c

Reply:
I don't think a convention around the order of the sections adds much value other than a visual consistency across the CHANGELOG file. However, I think @apps4everyone's point about **breaking** being at the top is an important one because they catch your attention better.